shared/testutil/spectest.go

```diff
@@ -29,6 +29,12 @@ var json = jsoniter.Config{
 	TagKey:                 "spec-name",
 }.Froze()
 
+// The spec BLS setting for the following test has been changed to 2 which is "BLS ignore", it's dependent on BLS being off in:
+// https://github.com/ethereum/eth2.0-specs/pull/1803
+// The new data for this test has not been resinged. It's recommended in 1814 to skip BLS verify for the following path:
+// https://github.com/ethereum/eth2.0-specs/issues/1841
+var skipBlsVerify = "success_surround"
```

Comment:
I believe this was fixed in the latest spec test. We should unskip it for v0.12.1

Reply:
Done